addon/services/file-manager.js

```diff
@@ -141,25 +244,65 @@ export default Ember.Service.extend({
                 headers: {
                     Authorization: `Bearer ${accessToken}`
                 },
-                success: (data) => resolve(data),
-                error: (_, __, error) => reject(error)
+                error: (_, __, error) => reject(error),
+                success: (data) => resolve(data)
             });
         });
     },
 
-    _pushToStore(data, id, parentFolder) {
-        // Hack the Waterbutler entity to look like the file Ember expects
-        data.data.id = id;
-        data.data.type = 'file';
-        let attr = data.data.attributes;
-        attr.materializedPath = attr.materialized;
-        attr.dateModified = attr.modified;
+    /**
+     * Get an updated `file` model from the OSF API.
+     *
+     * @method _reloadModel
+     * @private
+     * @param {Object} model Ember model to reload.
+     * @param {Function} isFresh Function that is called with the reloaded
```

Comment:
Minor nitpick, but can we rename this to something that sounds more like a function, and less like a boolean?